ig/input/fsh/VaccinationScheme.fsh

```diff
@@ -6,9 +6,56 @@ Description: """
   VaccinationScheme models a process consisting of immunizations through wich a patient can reach immunity against a target
   disease.
 """
-* extension contains VPVaccinationSchemeExtension named vaccinationSchemeExtension 1..1
+* extension contains VPVaccinationSchemeExtension named vaccinationSchemeExtension 1..1 MS
 
 Extension: VPVaccinationSchemeExtension
 Id: vp-vaccination-scheme-extension
 Title: "VaccinationSchemeExtension"
 Description: ""
+* extension contains
+  name 1..1 MS and
+  type 1..1 MS and
+  isPreferred 1..1 MS and
+  ageStart 0..1 MS and
+  ageEnd 0..1 MS
```

Comment:
There should be a mandatory reference to VPMedication.

Reply:
Addressed at 2f3e1a4